### ISSUE TITLE
Loop through action types and display result for all actions

### ISSUE THREAD

**Issue by chiragw15** [fossasia/susi_android#664]
**Actual Behaviour**

Query "multi3". The sever response contains 3 action types but only result of first is displayed.
Also for map type only first action type is used. 

**Expected Behaviour**
```
"actions": [
      {
        "type": "answer",
        "expression": "Singapore is a place with a population of 3547809. Here is a map: https://www.openstreetmap.org/#map=13/1.2896698812440377/103.85006683126556"
      },
      {
        "type": "anchor",
        "link": "https://www.openstreetmap.org/#map=13/1.2896698812440377/103.85006683126556",
        "text": "Link to Openstreetmap: Singapore"
      },
      {
        "type": "map",
        "latitude": "1.2896698812440377",
        "longitude": "103.85006683126556",
        "zoom": "13"
      }
    ]
```
If response is of this type, then the client should loop through all the actions and display result one after other with delay provided in the response. So, ideally the result of this response should be :
* 1st message : Singapore is a place with a population of 3547809. Here is a map: https://www.openstreetmap.org/#map=13/1.2896698812440377/103.85006683126556
* 2nd message after 0 delay : [Link to Openstreetmap: Singapore](https://www.openstreetmap.org/#map=13/1.2896698812440377/103.85006683126556)
* 3rd message after 0 delay: A map displaying the given coordinates and zoom level 

For response this 
```
"actions": [
      {
        "type": "answer",
        "expression": "Line 1"
      },
      {
        "type": "answer",
        "delay": 400,
        "expression": "Line 2"
      },
      {
        "type": "answer",
        "delay": 1000,
        "expression": "Line 3"
      }
    ]
```
Response should be 
* 1st message Line 1
* 2nd message after 0.4 seconds of 1st message : Line 2
* 3rd message after 1 second of 1st message : Line 3

**Comment by chiragw15** [comment]
This issue was getting a little bigger, so made sub issues for it #677 #678 #679 . Also sent a starting PR for this issue which adds loop to display result of all action types. Will be adding more commits to it to fix the above mentioned issue.

**Comment by chiragw15** [comment]
Fixed in #681
